packages/vue-client/src/assets/main.css

```diff
@@ -1,43 +1,96 @@
 @import "./base.css";
 
 #app {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-
   font-weight: normal;
+  display: grid;
+  grid-template-rows: max-content 1fr;
+  height: 100%;
 }
 
-@media (max-width: 540px) {
-  #app {
-    padding: 0.2rem;
+nav {
```

Comment:
I think this belongs inside the `<style>` of the `App.vue` component, because that's where the `<nav>` is declared too. Is there any reason to have it in the global `main.css`?

Reply:
Moved the nav style class into App.vue
The style in App.vue is scoped, so we left the combined style selectors that are also needed in UserNav in main.css
Thank you for the suggestion.